### ISSUE TITLE
[Issue]: When the logs folder is deleted, the script does not create another one automatically

### ISSUE THREAD

**Issue by rDamascena** [MichaelYochpaz/iSubRip#67]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Windows

### Python Version

3.10.11

### Package Version

2.5.3

### Description

When the logs folder is deleted, the script does not create another one automatically and returns an error.

I updated the script, deleted the configuration file and the old logs folder and it returned the error mentioned, which is fixed by creating a logs folder manually.

### Output Log

```Text
$ isubrip "https://tv.apple.com/tr/movie/sisu/umc.cmc.3mw4pbau8v6o6bxcujlnqz1r"
Error: [Errno 2] No such file or directory: 'C:\\Users\\Adriana\\.isubrip\\logs\\isubrip_2024-04-11_12-51-37.log'
```

**Comment by MichaelYochpaz** [comment]
Hey, thank you for reporting :)
Pushed a fix on e542923eaa1ef8455a52a4559765717282889d73, which will be included in the next version release.

**Comment by p1nkyy** [comment]
> Hey, thank you for reporting :) Pushed a fix on [e542923](https://github.com/MichaelYochpaz/iSubRip/commit/e542923eaa1ef8455a52a4559765717282889d73), which will be included in the next version release.

Hi how to apply this fix now before the next official release? I can't download sub due to this error. Thank you before.

**Comment by MichaelYochpaz** [comment]
The fix is included in the latest [v2.5.4](https://github.com/MichaelYochpaz/iSubRip/releases/tag/v2.5.4) release.
